src/app/content/components/Page/highlightManager.spec.tsx

```diff
@@ -1,21 +1,33 @@
 import UntypedHighlighter, {
   SerializedHighlight as UntypedSerializedHighlight
 } from '@openstax/highlighter';
-import { HTMLElement } from '@openstax/types/lib.dom';
+import { HTMLElement, MediaQueryList } from '@openstax/types/lib.dom';
 import defer from 'lodash/fp/defer';
 import keyBy from 'lodash/fp/keyBy';
 import React from 'react';
+import { Provider } from 'react-redux';
 import renderer from 'react-test-renderer';
+import createTestStore from '../../../../test/createTestStore';
 import { page } from '../../../../test/mocks/archiveLoader';
 import createMockHighlight from '../../../../test/mocks/highlight';
-import { assertWindow } from '../../../utils';
+import { Store } from '../../../types';
+import * as utils from '../../../utils';
 import Card from '../../highlights/components/Card';
-import CardWrapper from '../../highlights/components/CardWrapper';
+import { CardWrapper } from '../../highlights/components/CardWrapper';
 import { HighlightData } from '../../highlights/types';
 import highlightManager from './highlightManager';
 import { HighlightProp, stubHighlightManager } from './highlightManager';
 
 jest.mock('@openstax/highlighter');
+jest.mock('../../../utils', () => ({
+  ...jest.requireActual('../../../utils'),
+  assertWindow: () => ({
+    ...jest.requireActual('../../../utils').assertWindow(),
+    matchMedia: () => ({
+      matches: false,
+    } as MediaQueryList),
+  }),
+}));
```

Comment:
Does this affect only the `window` on line 56? If so then you could overwrite it there.